### PR TITLE
sst migration faq

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

**Review by randomJoe211 — DISMISSED** [review]

**Review by whitewum — APPROVED** [review]

File: docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ nebula> DROP TAG INDEX player_index_0;
 ## REBUILD INDEX
 
 ```ngql
-REBUILD {TAG | EDGE} INDEX <index_name> [OFFLINE]
+REBUILD {TAG | EDGE} INDEX <index_name> OFFLINE
 ```
 
 [Create Index](#create-index) section describes how to build indexes to improve query performance. If the index is created before inserting the data, there is no need to rebuild index and this section can be skipped; if data is updated or newly inserted before the index creation, it is necessary to rebuild the indexes in order to make sure that the indexes contain the previously added data. If the current database does not provide any services, use the `OFFLINE` keyword to speed up the rebuilding.
```
Comment (oldLady344) [inline docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/index.md:128]: The [Create Index]

"there is no need to rebuild index " ==> "there is no need to rebuild the index "

"If the index is created before inserting the data,  "==>"If the index is created before the data insertion,"

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md [hunk 2]
```diff
@@ -2,14 +2,16 @@
 
 The `FETCH` syntax is used to get vertex/edge's properties.
 
-## Fetch Vertex property
+## Fetch Vertex Property
```
Comment (oldLady344) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md:5]: Fetch Vertex Properties

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md [hunk 3]
```diff
@@ -2,14 +2,16 @@
...
 
 Use `FETCH PROP ON` to return a (list of) vertex's properties. Currently, you can get multiple vertices' properties with the same tag in one statement. You can use `FETCH` together with [pipe](../../3.language-structure/pipe-syntax.md) and [user defined variables](../../3.language-structure/user-defined-variables.md).
 
 ```ngql
 FETCH PROP ON {<tag_name_list> | *} <vertex_id_list> [YIELD [DISTINCT] <return_list>]
 ```
 
+Fetch `<tag_name_list>` is only available for version 1.0.1 or later version. If your **Nebula Graph** is earlier than 1.0.1, you can only fetch one type of vertices in a single query.
```
Comment (oldLady344) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md:13]: "Fetch `<tag_name_list>`" ==> "The <tag_name_list> parameter can work with the `FETECH` statement only in Nebula Graph 1.0.1 or later version"?

If your Nebula Graph version is earlier than 1.0.1?

File: docs/manual-EN/1.overview/2.quick-start/2.FAQ.md [hunk 4]
```diff
@@ -177,3 +177,7 @@ If max_edge_returned_per_vertex is set to be 10, when filtering with WHERE, how
 When using `FETCH` to return data, sometimes the corresponding data is returned but sometimes it doesn't work.
 
 If you encounter the above situation, please check if you have run two storage services on the same node and the port of the two storage service are the same. If so, please modify one of the storage port and import the data again.
+
+### Does the sst Files Support Migration
+
+The sst files used by the storaged is bound to the graph space in the cluster. Thus you can not copy the sst files directly to a new graph space or a cluster.
```
Comment (oldLady344) [inline docs/manual-EN/1.overview/2.quick-start/2.FAQ.md:183]: "or a cluster." ==> "or a new cluster."?

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**